driver-sync/src/examples/documentation/DocumentationSamples.java

```diff
@@ -516,6 +530,93 @@ public void apply(final Document document) {
         });
     }
 
+    @Test
+    public void testAggregate() {
+
+        MongoCollection<Document> salesCollection = database.getCollection("sales");
+
+        // Start Aggregation Example 1
+        AggregateIterable<Document> aggregateIterable = salesCollection.aggregate(asList(
+                match(eq("items.fruit", "banana")),
+                sort(ascending("date"))
+        ));
+        // End Aggregation Example 1
+
+        aggregateIterable.forEach(new Block<Document>() {
```

Comment:
The drivers ticket says not to test the output, just make sure it can execute.  Adding the forEach ensures the aggregate at least gets executed on the server, albeit with an empty stream of documents.

Reply:
Is there any point in this? aggregationIterable can execute - its not part of the docs example so seems pointless to execute.